core/embed/rust/src/ui/model_mercury/component/hold_to_confirm.rs

```diff
@@ -188,8 +188,8 @@ impl Component for HoldToConfirm {
     fn place(&mut self, bounds: Rect) -> Rect {
         self.area = bounds;
         self.button.place(Rect::snap(
-            self.area.center(),
-            Offset::uniform(120),
+            self.area.center() + Offset::y(9),
```

Comment:
Perhaps, use the `CONFIRM_ANIMATION_OFFSET` from target branch

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/c5abc8ad08b7f65bf22f7129a701b38c3c799b1c